cloud-service-providers/aws/eks/README.md

```diff
@@ -0,0 +1,179 @@
+# NIM-EKS: NVIDIA Inference Microservices on AWS EKS
+This GitLab repository is dedicated to testing NIM (NVIDIA Inference Microservices) on AWS EKS (Elastic Kubernetes Service).
+
+## AWS Infrastructure setup:
+
+1:Refer this high-level architecture diagram for an overview of the setup
+![High level architecture diagram](aws-eks-architecture.png)
+
+2: Refer this to prepare your local environment to run cdk
+https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html
+
+There are four stacks in the nim-eks-cdk/lib directory. You can either deploy all of them using `cdk deploy --all` or deploy each stack by specifying the stack name
+like `cdk deploy vpc-stack`
+
+Stack Deployment Order:
+
+    cdk deploy vpc-stack
+    cdk deploy eks-cluster-stack
+    cdk deploy efs-stack
+
+Note:  Ensure that you are in the nim-eks-cdk directory to run the above mentioned commands
+
+ClusterAdmin user is created by the cdk. Create access keys for this user to run kubectl and helm commands. Once secret credentials are created run the following command to manage the cluster
+
+    aws eks update-kubeconfig --name <eks-cluster-name> --region <region-code> --profile <profile-name-if-any>
+
+Note:  Ensure that you are in the cloud-service-providers/aws/eks directory for the rest of this guide.
+
+## Cluster setup for inference:
+
+1: Install NVIDIA Device Plugin: Install NVIDIA device plugins to run GPU workloads. Check CUDA base image version for compatibility.
```

Comment:
Is there a reason we are directly installing the device plugin and not the GPU Operator?

Reply:
This example uses Amazon EKS optimized Amazon Linux AMIs (Amazon Linux 2 AMI). It's default configuration isn't compatible with GPU operator. https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/amazon-eks.html